src/Model.php

```diff
@@ -428,34 +417,34 @@ public function init(): void
         }
 
         // Declare our basic CRUD actions for the model.
-        $this->addAction('add', [
+        $this->addUserAction('add', [
             'fields' => true,
-            'modifier' => UserAction\Generic::MODIFIER_CREATE,
-            'scope' => UserAction\Generic::NO_RECORDS,
+            'modifier' => Model\UserAction::MODIFIER_CREATE,
+            'scope' => Model\UserAction::SCOPE_NONE,
             'callback' => 'save',
             'description' => 'Add ' . $this->getModelCaption(),
             'ui' => ['icon' => 'plus'],
         ]);
-        $this->addAction('edit', [
+        $this->addUserAction('edit', [
             'fields' => true,
-            'modifier' => UserAction\Generic::MODIFIER_UPDATE,
-            'scope' => UserAction\Generic::SINGLE_RECORD,
+            'modifier' => Model\UserAction::MODIFIER_UPDATE,
+            'scope' => Model\UserAction::SINGLE_RECORD,
```

Comment:
here is `SINGLE_RECORD`, isn't is mixed with `SCOPE_`?

Do you have any ideas how to not use `SCOPE`? Wdyt about `APPLIES_TO_XXX` (so it is like `MODIFIER_`, we can use that prefix for property, ...)?